### PR TITLE
Update release script to use changelog script from #1235

### PR DESCRIPTION
**Pull request by avarsava**
JIRA Ticket: N/A

Already updated the wiki page for the shesmu release to reflect these changes

- [ ] Updates Changelog
- [ ] Updates developer documentation

### CODE REVIEW

**Review by callunity — APPROVED** [review]

**Review by djcooke — COMMENTED** [review]
This makes it impossible to do a major release or force a minor release when the changes would indicate it should be a patch release. MISO (original source of this changelog setup) uses a script argument for those options: https://github.com/miso-lims/miso-lims/blob/develop/release.sh#L21

**Comment by avarsava** [comment]
@djcooke thanks! fixed

File: release.sh [hunk 1]
```diff
@@ -22,26 +43,38 @@ fi
...
-if ! grep -q "# Unreleased" RELEASE_NOTES.md; then
-	echo "Error: changelog does not contain an Unreleased section"
-	exit 4
-fi
-mv RELEASE_NOTES.md RELEASE_NOTES.md.old
-sed "s/# Unreleased/# Unreleased\n\n# \[${NEW_VERSION}\] - $(date -u +%Y-%m-%dT%H:%M+00:00)/g" RELEASE_NOTES.md.old > RELEASE_NOTES.md
-rm RELEASE_NOTES.md.old
-${EDITOR:-editor} RELEASE_NOTES.md
+echo "Preparing ${RELEASE_TYPE} release ${NEW_VERSION}..."
+./compact-changelog.sh ${NEW_VERSION} || return 2
```
Comment (djcooke) [inline release.sh:77]:
```suggestion
./compact-changelog.sh ${NEW_VERSION} || exit 2
```
Sorry, missed this on first pass

Comment (avarsava) [reply release.sh]: Fixed!

**Review by djcooke — APPROVED** [review]

**Pull request merged**